tests/unit/test_interrogator.py

```diff
@@ -237,25 +237,33 @@ def test_is_image_visible_by_javascript_svg_no_src():
 
 
 def test_is_element_visible():
+    mock_driver = MagicMock()
     elements = [
         MockElement("true")
     ]
     finder = MockFinder(list_of_elements_to_return=elements)
-    test_interrogator = Interrogator("", finder, "logger")
+    wait_time = 10
+    test_interrogator = Interrogator(mock_driver, finder, wait_time, "logger")
 
     test_interrogator.is_element_visible(PageElement(By.ID, "some_id"))
+    assert_that(mock_driver.implicitly_wait.call_count, equal_to(2), "Expected two calls to implicitly_wait")
+    assert mock_driver.implicitly_wait.called, "Expected implicitly_wait to be called"
```

Comment:
This line is not needed because you have already checked for two calls to implicitly_wait on the line above.